use-cases/model-fine-tuning-pipeline/data-preparation/gemma-it/src/dataprep.py

```diff
@@ -197,53 +190,77 @@ def extract_product_details(text):
 )
 def generate_content(context):
     try:
```

Comment:
Do you want it to fail for these exceptions? They aren't in the tenacity retry block.

Reply:
we haven't encountered the errors yet

Reply:
Alright, just know that it will fail with an exception for those raises if they aren't included in the retry.

Reply:
yup, haven't experienced as we don't know if it's something we want to retry yet

Reply:
just hit one, RateLimit